bitnami/scylladb/values.yaml

```diff
@@ -0,0 +1,1115 @@
+      ##
+      adaptSecurityContext: auto
+## @section Common parameters
+##
+
+## @param nameOverride String to partially override common.names.fullname
+##
+nameOverride: ""
+## @param fullnameOverride String to fully override common.names.fullname
+##
+fullnameOverride: ""
+## @param kubeVersion Force target Kubernetes version (using Helm capabilities if not set)
+##
+kubeVersion: ""
+## @param commonLabels Labels to add to all deployed objects (sub-charts are not considered)
+##
+commonLabels: {}
+## @param commonAnnotations Annotations to add to all deployed objects
+##
+commonAnnotations: {}
+## @param clusterDomain Kubernetes cluster domain name
+##
+clusterDomain: cluster.local
+## @param extraDeploy Array of extra objects to deploy with the release
+##
+extraDeploy: []
+## Enable diagnostic mode in the deployment
+##
+diagnosticMode:
+  ## @param diagnosticMode.enabled Enable diagnostic mode (all probes will be disabled and the command will be overridden)
+  ##
+  enabled: false
+  ## @param diagnosticMode.command Command to override all containers in the deployment
+  ##
+  command:
+    - sleep
+  ## @param diagnosticMode.args Args to override all containers in the deployment
+  ##
+  args:
+    - infinity
+## @section Scylladb parameters
+##
+
+## Bitnami Scylladb image
+## ref: https://hub.docker.com/r/bitnami/scylladb/tags/
+## @param image.registry [default: REGISTRY_NAME] Scylladb image registry
+## @param image.repository [default: REPOSITORY_NAME/scylladb] Scylladb image repository
+## @skip image.tag Scylladb image tag (immutable tags are recommended)
+## @param image.digest Scylladb image digest in the way sha256:aa.... Please note this parameter, if set, will override the tag
+## @param image.pullPolicy image pull policy
+## @param image.pullSecrets Scylladb image pull secrets
+## @param image.debug Enable image debug mode
+##
+image:
+  registry: docker.io
+  # TODO: Change the image once it is published
+  repository: javsalgar/scylla-test
+  tag: 11
+  digest: ""
+  ## Specify a imagePullPolicy
+  ## Defaults to 'Always' if image tag is 'latest', else set to 'IfNotPresent'
+  ## ref: https://kubernetes.io/docs/concepts/containers/images/#pre-pulled-images
+  ##
+  pullPolicy: IfNotPresent
+  ## Optionally specify an array of imagePullSecrets.
+  ## Secrets must be manually created in the namespace.
+  ## ref: https://kubernetes.io/docs/tasks/configure-pod-container/pull-image-private-registry/
+  ## e.g:
+  ## pullSecrets:
+  ##   - myRegistryKeySecretName
+  ##
+  pullSecrets: []
+  ## Enable debug mode
+  ##
+  debug: false
+
+## Database credentials
+## @param dbUser.user Scylladb admin user
+## @param dbUser.forcePassword Force the user to provide a non
+## @param dbUser.password Password for `dbUser.user`. Randomly generated if empty
+## @param dbUser.existingSecret Use an existing secret object for `dbUser.user` password (will ignore `dbUser.password`)
+##
+dbUser:
+  user: cassandra
```

Comment:
cassandra?

Reply:
Yes, it's what upstream scylladb does, in order to maintain a similar UX as Cassandra